### PR TITLE
added DirectorDao test

### PR DESCRIPTION
**Pull request by phoenixday**
there're classes Director, DirectorDao, DirectorDaoImpl, but I added them just for tests, no implementation :)

### CODE REVIEW

File: persistence/src/main/java/cz/muni/fi/pa165/entity/Director.java [hunk 1]
```diff
@@ -15,4 +15,24 @@
...
+    }
+
+    public Director() {
+
+    }
+
+    public void setName(String fullName) {
+    }
+
+    public void store(Director d) {
```
Comment (xjurko) [inline persistence/src/main/java/cz/muni/fi/pa165/entity/Director.java:28]: should store be defined on an entity itself ?

File: persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ * @author alia
+ */
+
+@Ignore("enable when director implemented")
+@ContextConfiguration(classes = PersistenceConfig.class)
+@TestExecutionListeners(TransactionalTestExecutionListener.class)
+@Transactional
+public class DirectorDaoTest extends AbstractTestNGSpringContextTests {
+
+    @Autowired
```
Comment (xjurko) [inline persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java:35]: better to use `@Inject`

File: persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        assertNotNull(found);
+        assertEquals(Optional.of(d1), found);
+    }
+
+    @Test
+    public void testRemove() {
+        Director d1 = new Director("d1");
+        directorDao.store(d1);
+        assertNotNull(directorDao.findById(d1.getId()));
```
Comment (xjurko) [inline persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java:83]: also need assert that the found director is not Optional.empty

File: persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        directorDao.remove(d1);
+        assertNull(directorDao.findById(d1.getId()));
```
Comment (xjurko) [inline persistence/src/test/java/cz/muni/fi/pa165/dao/DirectorDaoTest.java:85]: If director is not in DB it will not be null but Optional.empty

**Review by xjurko — CHANGES_REQUESTED** [review]

**Review by xjurko — APPROVED** [review]
lgtm

**Review by lsolodkova — APPROVED** [review]

**Pull request merged**